### DISCUSSION TITLE
Which 🟩 is usually found in a Caesar 🥗?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1003] [Galaxy Trivia] [answered]
**Bonus:** What are the 5 mother sauces?

**Comment by Sriansh-raj** [comment]
The Caesar salad usually contains romaine lettuce, croutons, Parmesan cheese, and Caesar dressing. The dressing is typically made with ingredients such as garlic, Dijon mustard, Worcestershire sauce, lemon juice, olive oil, and egg yolks.
The five mother sauces are the foundation of French cuisine and include:
- Béchamel: a white sauce made with milk and a white roux (butter and flour)
- Velouté: a light-colored sauce made with stock and a blond roux (butter and flour)
- Espagnole: a brown sauce made with browned mirepoix (onions, carrots, and celery), browned flour, and brown stock
- Hollandaise: an emulsified sauce made with egg yolks, butter, and lemon juice
- Tomato: a sauce made with tomatoes, mirepoix, and a roux.

**Comment by peterbaccaro** [comment]
The most common green leafy vegetable found in a Caesar salad is **romaine lettuce**. Romaine lettuce is characterized by its long, crisp leaves and is often used as the base of a Caesar salad. Other common ingredients in a Caesar salad include Parmesan cheese, croutons, Caesar dressing (which typically includes ingredients like garlic, anchovies, lemon juice, Dijon mustard, and olive oil), and sometimes additional toppings like grilled chicken or bacon.

The concept of the five mother sauces comes from classical French cuisine. These sauces are considered the foundation for many other sauces and variations. The five mother sauces are:

- **Béchamel Sauce**: This sauce is made from a white roux (cooked flour and butter) mixed with milk. It is commonly used as a base for creamy dishes, such as macaroni and cheese or gratins.

- **Velouté Sauce**: Velouté is a sauce made from a light roux (cooked flour and butter) combined with a clear stock, such as chicken, veal, or fish stock. It forms a smooth, velvety sauce that can be used with various ingredients and flavors.

- **Espagnole Sauce**: Also known as brown sauce, espagnole is a rich and dark sauce made from browned mirepoix (a mixture of diced onions, carrots, and celery), browned flour, tomato paste, and brown stock. It is often used as a base for other sauces or to enhance meat dishes.

- **Tomato Sauce**: Tomato sauce is a classic sauce made from tomatoes, onions, garlic, and various herbs and spices. It can be seasoned and flavored in different ways to create variations such as marinara sauce or arrabbiata sauce.

- **Hollandaise Sauce**: Hollandaise is an emulsion sauce made by whisking melted butter into egg yolks, resulting in a rich and creamy sauce. It is typically flavored with lemon juice and used to accompany dishes like eggs Benedict or steamed vegetables.

**Comment by thuongtruong1009** [comment, marked as answer]
The green vegetable typically found in a Caesar salad is romaine lettuce. 

The five mother sauces, which form the foundation of classical French cuisine, are:

1. Béchamel Sauce: This sauce is made from a roux (a mixture of flour and butter) and milk. It is commonly used in dishes like macaroni and cheese or as a base for other sauces.

2. Velouté Sauce: Velouté is a sauce made from a roux and a light chicken, veal, or fish stock. It provides a creamy and velvety texture and is often used with poultry or seafood dishes.

3. Espagnole Sauce: Espagnole, also known as brown sauce, is a rich and flavorful sauce made from a brown roux and a dark stock, often beef or veal. It is typically used in dishes like braised meats or stews.

4. Tomato Sauce: Tomato sauce, as the name suggests, is made primarily from tomatoes. It can be a simple sauce or include additional ingredients such as onions, garlic, herbs, and spices. Tomato sauce is a versatile base used in various Italian dishes.

5. Hollandaise Sauce: Hollandaise is an emulsion sauce made by whisking clarified butter into egg yolks, resulting in a smooth and rich sauce. It is often served with eggs Benedict or vegetables.

These mother sauces serve as a basis for creating numerous derivative sauces by incorporating additional ingredients, herbs, spices, or reductions.
